assets/javascripts/kitten/components/form/select-with-state.js

```diff
@@ -38,14 +38,14 @@ export class SelectWithState extends Component {
 
   onKeyDown(event) {
     const {
-      enterKeyCode,
-      spaceKeyCode,
+      enterKeyCode = 13,
+      spaceKeyCode = 32,
```

Comment:
Ici la syntaxe à utiliser c'est vraiment `const enterKeyCode = 13`, et ne pas utiliser l'extraction de variables de `this.props`. Je suis étonné que ça fonctionne d'ailleurs oO.